### ISSUE TITLE
Remove warnings from npm install step

### ISSUE THREAD

**Issue by helio-frota** [trustification/trustify-ui#17]
```
➜  trustify-ui git:(main) ✗ npm i
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm WARN deprecated domexception@4.0.0: Use your platform's native DOMException instead
```

* npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated
* npm WARN deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
* npm WARN deprecated domexception@4.0.0: Use your platform's native DOMException instead

**Comment by carlosthe19916** [comment]
What problem are we solving removing the warnings? I am not a fan of wasting time on tasks that add no value to the app. Unless we identity a bug or a real problem I do not think we should invest a minute on it.

**Comment by helio-frota** [comment]
I understand your point but currently we can't just update the deps without breaking ts compilation and other webpack parts.

So I'm splitting the problem in small parts :+1:

**Comment by ctron** [comment]
> What problem are we solving removing the warnings?

We don't seem to know, right? However, ignoring warnings feels like bad strategy. Someone might have a reason for "warning" others. Go going ahead and cleaning up warnings is an important task IMHO.

Cleaning up can come in two ways: replacing the dependency, or figuring out that there's no harm. In the latter case, the warning should be silenced. If technically not possible, there should be a note somewhere.

Personally I find builds with warnings quite distracting and suspicious.

**Comment by helio-frota** [comment]
> Someone might have a reason for "warning" others

yeah I found some things already... and that is not only related to update the version numbers.. some of them are dropping support for `node < 18` and etc... 

similar to this task https://gist.github.com/helio-frota/dab8edd2b1862bcbb6567e2cb1d29e1e#http where we need to read the change-log-release docs , build, run etc etc .

imo, its not a good thing to spent hours (days?) in future ~6 months trying to fix an "unfixable" because good amount of breaking changes...

> Personally I find builds with warnings quite distracting and suspicious.

+1

**Comment by helio-frota** [comment]
closing stale issue 👍
